docs/data/data-grid/export/export.md

```diff
@@ -390,13 +390,6 @@ setupExcelExportWebWorker({
 
 ## 🚧 Clipboard [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
```

Comment:
Added a link to the new page: https://deploy-preview-7389--material-ui-x.netlify.app/x/react-data-grid/export/#clipboard
Also, removed the Premium plan icon, since clipboard copy is not a premium feature